### ISSUE TITLE
Edit an existing idea

### ISSUE THREAD

**Issue by fbrnc** [ivandiazwm/phpback#99]
Is it not possible to edit an idea that the user has created or am I just missing something? :)

**Comment by peterhuerlimann** [comment]
No, to the best of my knowledge this is not possible currently.

**Comment by dosch** [comment]
+1 for this feature request.
